specification/voiceservices/data-plane/Microsoft.VoiceServices/stable/2023-10-01/swagger.json

```diff
@@ -0,0 +1,1015 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft.VoiceServices.ACG.ProvisioningPlatform.API",
+    "version": "2023-10-01"
+  },
+  "securityDefinitions": {
+    "OAuth2": {
+      "type": "oauth2",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "flow": "application",
+      "tokenUrl": "https://login.microsoftonline.com/common/oauth2/v2.0/token",
+      "scopes": {
+        "https://func-voiceservice-rp-prod-eastuseuap.azurewebsites.net/.default": "Grants the associated roles assigned to the client application"
+      }
+    }
+  },
+  "security": [
+    {
+      "OAuth2": [
+        "https://func-voiceservice-rp-prod-eastuseuap.azurewebsites.net/.default"
+      ]
+    }
+  ],
+  "paths": {
+    "/account/{accountName}/numbers": {
+      "get": {
+        "operationId": "Account_GetNumbers",
```

Comment:
This operation and several others are missing the required "api-version" query parameter.
```
>spectral lint $f | grep api-version
  32:22      warning  az-version-policy                Operation does not define an "api-version" query parameter.                paths./account/{accountName}/numbers.get.parameters
  98:22      warning  az-version-policy                Operation does not define an "api-version" query parameter.                paths./account/{accountName}/numbers/batch.post.parameters
 154:22      warning  az-version-policy                Operation does not define an "api-version" query parameter.                paths./account/{accountName}/numbers/{phoneNumber}.delete.parameters
 208:22      warning  az-version-policy                Operation does not define an "api-version" query parameter.                paths./account/{accountName}/numbers/{phoneNumber}.get.parameters
 268:22      warning  az-version-policy                Operation does not define an "api-version" query parameter.                paths./account/{accountName}/numbers/{phoneNumber}.put.parameters
 344:22      warning  az-version-policy                Operation does not define an "api-version" query parameter.                paths./accounts.get.parameters
 397:22      warning  az-version-policy                Operation does not define an "api-version" query parameter.                paths./accounts/{accountName}.delete.parameters
 441:22      warning  az-version-policy                Operation does not define an "api-version" query parameter.                paths./accounts/{accountName}.get.parameters
 499:22      warning  az-version-policy                Operation does not define an "api-version" query parameter.                paths./accounts/{accountName}.put.parameters
 565:22      warning  az-version-policy                Operation does not define an "api-version" query parameter.                paths./numbers.get.parameters
```

Reply:
Ah, apologies - I'd created the parameter description but not included it on the endpoints - fixed.